app/gui/CliStartStreamSegue.qml

```diff
@@ -0,0 +1,78 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.2
+import QtQuick.Dialogs 1.2
+
+import ComputerManager 1.0
+
+Item {
+    visible: false
+
+    function onSearchingComputer(name) {
+        stageLabel.text = "Searching computer " + name + "..."
```

Comment:
I think we could just use a generic message here like: `"Establishing connection to PC..."`

Reply:
Ok 👍